app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,185 @@
+<resources>
+    <string name="error_generic">Произошла ошибка.</string>
+    <string name="error_empty">Не может быть пустым.</string>
+    <string name="error_invalid_domain">Некорректный домен.</string>
+    <string name="error_failed_app_registration">Ошибка аутентификации на этом узле.</string>
+    <string name="error_no_web_browser_found">Не удается найти веб-браузер.</string>
+    <string name="error_authorization_unknown">Произошла ошибка неопознанной авторизации.</string>
+    <string name="error_authorization_denied">Авторизация была отклонена.</string>
+    <string name="error_retrieving_oauth_token">Не удалось получить токен авторизации.</string>
+    <string name="error_compose_character_limit">Статус слишком длинный!</string>
+    <string name="error_media_upload_size">Файл должен быть не больше 4 Мбайт.</string>
+    <string name="error_media_upload_type">Данный тип файла не может быть загружен.</string>
+    <string name="error_media_upload_opening">Файл не может быть открыт.</string>
+    <string name="error_media_upload_permission">Необходимо разрешение на чтение медиаконтента.</string>
+    <string name="error_media_download_permission">Необходимо разрешение для хранения медиаконтента.</string>
+    <string name="error_media_upload_image_or_video">Изображения и видео не могут быть прикрекплены к статусу одновременно.</string>
+    <string name="error_media_upload_sending">Загрузка не удалась.</string>
+    <string name="error_report_too_few_statuses">Должен быть выбран хотя бы один статус.</string>
+
+    <string name="title_home">Дом</string>
+    <string name="title_notifications">Уведомления</string>
+    <string name="title_public_local">Локальная лента</string>
+    <string name="title_public_federated">Глобальная лента</string>
+    <string name="title_thread">Тред</string>
+    <string name="title_tag">#%s</string>
+    <string name="title_statuses">Посты</string>
+    <string name="title_follows">Подписки</string>
+    <string name="title_followers">Подписчики</string>
+    <string name="title_favourites">Понравившееся</string>
+    <string name="title_mutes">Список глушения</string>
```

Comment:
@charlag I took this from Russian translation of Mastodon (for compatibility reasons).

![](http://i.imgur.com/hBDt7Qy.png)